.github/workflows/main.yaml

```diff
@@ -11,15 +11,20 @@ jobs:
       matrix:
         os: [ubuntu-latest]
 
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
With actions enabled for PRs (and PRs from forked repos), you can use the built in token `secrets.GITHUB_TOKEN`, see https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Reply:
Got it! Thanks to let me know.